.github/actions/spell-check/expect.txt

```diff
@@ -14,18 +14,17 @@ Afek
 afl
 afnic
 afsdb
-AFYER
 agentx
 agentxperms
+aggr
 Aguiar
 ahupowerdns
 Aips
 Aki
+AKq
```

Comment:
I think this is from mailarchive.ietf.org ... there is a url pattern that would match it, but it was disabled and I preserved that in this refresh.